class ContainerError(Exception):
    pass


class ValidationError(Exception):
    pass

